### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hs26gill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,3 +68,5 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
 ### `yarn build` fails to minify
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
```
Comment (hs26gill) [inline README.md:71]:
```suggestion
please fix
```

File: README.md [hunk 2]
```diff
@@ -68,3 +68,5 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
 ### `yarn build` fails to minify
```
Comment (hs26gill) [inline README.md:69]:
```suggestion
fix this
```

File: README.md [hunk 3]
```diff
@@ -68,3 +68,5 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
...
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
```
Comment (angad-singh) [inline README.md:70]: Yeah agree with @hs26gill

File: README.md (hunk 2 above)
Comment (angad-singh) [reply README.md:69]: 👍🏼

**Pull request closed without merging**